content/docs/setup/kubernetes/multicluster-install.md

```diff
@@ -91,18 +131,24 @@ plane. Creating secrets will register the secrets with Istio properly.
 
 > The local cluster running the Istio control plane does not need
 it's secrets stored and labeled. The local node is always aware of
-it's Kubernetes credentials, but the local node is not aware of
+its Kubernetes credentials, but the local node is not aware of
 the remote nodes' credentials.
 
 Create a secret and label it properly for each remote cluster:
 
 ```command
-$ pushd $HOME/multicluster
+$ pushd $WORK_DIR
```

Comment:
My VM do not have `pushd`, is it possible we just use `cd` instead?

Reply:
I'll change it.  Was just keeping with the prior version but pushd isn't specifically needed